src/Microsoft.Diagnostics.Monitoring.RestServer/StreamingLogger.cs

```diff
@@ -84,40 +84,47 @@ private void LogJson<TState>(LogLevel logLevel, EventId eventId, TState state, E
             //CONSIDER Should we cache up the loggers and writers?
             using (var jsonWriter = new Utf8JsonWriter(outputStream, new JsonWriterOptions { Indented = false }))
             {
+                // Matches the format of JSONConsoleFormatter
+
                 jsonWriter.WriteStartObject();
+                // TODO: Write timestamp as string
+                //jsonWriter.WriteString("Timestamp", DateTime.UtcNow.ToString());
                 jsonWriter.WriteString("LogLevel", logLevel.ToString());
-                jsonWriter.WriteString("EventId", eventId.ToString());
+                jsonWriter.WriteNumber("EventId", eventId.Id);
                 jsonWriter.WriteString("Category", _categoryName);
+                jsonWriter.WriteString("Message", formatter(state, exception));
                 if (exception != null)
                 {
-                    jsonWriter.WriteString("Exception", formatter(state, exception));
-                }
-                else
-                {
-                    jsonWriter.WriteString("Message", formatter(state, exception));
+                    jsonWriter.WriteString("Exception", exception.ToString());
```

Comment:
Did you want to change the text format to produce exception verbose information as well? Currently it always uses the formatter.

Reply:
JsonConsoleFormatter uses `formatter(state, exception` for the `Message` property only and just calls `ToString` on the exception for the `Exception` property.

Reply:
To clarify, I meant add another data: <exception.ToString()> to the text/event-stream format.

Reply:
Sure, I can do that. This change is only about the JSON format. I'll update event-stream next.

Reply:
I added the event-stream update since it was simple.